File: backend/employer-profile-service/src/main/java/com/jobportal/employerprofile/dto/EmployerProfileResponseDto.java
package com.jobportal.employerprofile.dto;

public class EmployerProfileResponseDto {

    private Integer employerId;
    private Integer userId;
    private String companyName;
    private String email;
    private String address;
    private Integer city;
    private Integer state;
    private String country;
    private String registrationId;
    private String description;
    private String industry;

    public Integer getEmployerId() { return employerId; }
    public void setEmployerId(Integer employerId) { this.employerId = employerId; }
    public Integer getUserId() { return userId; }
    public void setUserId(Integer userId) { this.userId = userId; }
    public String getCompanyName() { return companyName; }
    public void setCompanyName(String companyName) { this.companyName = companyName; }
    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }
    public String getAddress() { return address; }
    public void setAddress(String address) { this.address = address; }
    public Integer getCity() { return city; }
    public void setCity(Integer city) { this.city = city; }
    public Integer getState() { return state; }
    public void setState(Integer state) { this.state = state; }
    public String getCountry() { return country; }
    public void setCountry(String country) { this.country = country; }
    public String getRegistrationId() { return registrationId; }
    public void setRegistrationId(String registrationId) { this.registrationId = registrationId; }
    public String getDescription() { return description; }
    public void setDescription(String description) { this.description = description; }
    public String getIndustry() { return industry; }
    public void setIndustry(String industry) { this.industry = industry; }
}

File: backend-dotnet/Services/JobPortal.EmployerProfileService/DTOs/EmployerProfileResponseDto.cs
namespace JobPortal.EmployerProfiles.DTOs;

public class EmployerProfileResponseDto
{
    public int EmployerId { get; set; }
    public int UserId { get; set; }
    public string CompanyName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string? Address { get; set; }
    public int City { get; set; }
    public int State { get; set; }
    public string Country { get; set; } = string.Empty;
    public string RegistrationId { get; set; } = string.Empty;
    public string? Description { get; set; }
    public string Industry { get; set; } = string.Empty;
}
